src/bindings/js/node/lib/addon.ts

```diff
@@ -13,6 +13,7 @@ type SupportedTypedArray =
 
 interface Core {
   compileModel(model: Model, device: string): CompiledModel;
```

Comment:
Why async version doesn't have config parameter?
I think it is a mistake, because tests applies config.

Reply:
I must have forgotten, thanks ;)